app/controllers/accounts_controller.rb

```diff
@@ -117,41 +120,62 @@ def update
     account = current_organization.accounts.find(params[:id])
     return forbidden unless account && current_account.allowed_to_edit_account?(account, current_organization)
     return json(account) unless account.update_attributes pick(params, :first_name, :last_name, :email,:language, :document_language)
+
     role = pick(params, :role)
     #account.update_attributes(role) if !role.empty? && current_account.admin?
     membership = current_organization.role_of(account)
     membership.update_attributes(role) if !role.empty? && current_account.admin?
     password = pick(params, :password)[:password]
     if (current_account.id == account.id) && password
-      account.password = password
-      account.save
+      account.update(password: password)
     end
-    json account.canonical( :membership=>membership )
+    json account.canonical(membership: membership)
   end
 
   # Resend a welcome email for a pending account.
   def resend_welcome
     return forbidden unless current_account.admin?
+
     account = current_organization.accounts.find(params[:id])
-    LifecycleMailer.login_instructions( account, current_organization, current_account ).deliver_now
+    LifecycleMailer.login_instructions(account, current_organization, current_account).deliver_now
     json nil
   end
   
   def mailboxes
-    # code goes here.
+    # TODO: Filter by status, once that exists
+    @mailboxes = UploadMailbox.where(membership_id: @current_account.memberships.pluck(:id))
+    set_minimal_nav text: 'Back to workspace', link: '/search/'
   end
```

Comment:
Good question. I went with `@current_account` because it's set via this path:

1. [`before_action :login_required`](https://github.com/documentcloud/documentcloud/blob/49f7a07032ee478353a536b2c59305f93a76c7f1/app/controllers/accounts_controller.rb#L7)
2. [ApplicationController#logged_in?](https://github.com/documentcloud/documentcloud/blob/53c5e293ddacaf88704811f511dee67ca65af4e3/app/controllers/application_controller.rb#L175-L177)
3. [ApplicationController#current_account](https://github.com/documentcloud/documentcloud/blob/53c5e293ddacaf88704811f511dee67ca65af4e3/app/controllers/application_controller.rb#L228-L232)

But elsewhere in the controller, we use `current_account` directly. I prefer @current_account, but I suppose it's less safe? @knowtheory any preference?